src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -70,12 +70,12 @@ namespace OpenMS
   {
 public:
 
-    inline static ModificationsDB * getInstance()
+    inline static ModificationsDB * getInstance(OpenMS::String unimod_file = "CHEMISTRY/unimod.xml", OpenMS::String psimod_file = "CHEMISTRY/PSI-MOD.obo")
     {
       static ModificationsDB * db_ = 0;
       if (db_ == 0)
       {
-        db_ = new ModificationsDB;
+        db_ = new ModificationsDB(unimod_file, psimod_file);
```

Comment:
also it means all later calls to get instance will get the initial db, no matter the parameters

@grosenberger this kind of counter-intuitive behavior needs to be documented, otherwise I am sure people will be surprised later on